libyul/CompilabilityChecker.cpp

```diff
@@ -48,8 +48,8 @@ map<YulString, int> CompilabilityChecker::run(
 
 		BuiltinContext builtinContext;
 		builtinContext.currentObject = &_object;
-		for (auto name: _object.dataNames())
-			builtinContext.subIDs[name] = 1;
+		for (auto const& subNode: _object.subObjects)
+			builtinContext.subIDs[subNode->name] = 1;
```

Comment:
Another difference (not sure how much it actually affects this code) is that `dataNames()` was filtering out empty names and adding the name of the current object. That's not the case with `subObjects`.

Reply:
Just in case, I added the name of current object